CLIENTS.md

```diff
@@ -0,0 +1,75 @@
+# Featured Client Requirements
+Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
+
+## Terms/Definitions
+
+- Us/We - The Meower team
+
+## Rules
+- Clients created for malicious intent will not be featured
+- If your client implements any kind of plugin/addon interface, any plugins that are featured (or in the repo) are considered to be part of the client.
+- The Meower Team may, at it's sole discretion, choose whether or not a client should be featured
+- The Meower Team may feature a client despite not complying with this RFC
+
+## Guidelines
+
+### Design Practices
+
+- Cache chat and post data
+- Use Cloudlink to get updates to cached information
+- You should use the latest version of the API
+
+### Vulnerabilities
+
+If your client is discovered to have a vulnerability, you must fix said vulnerability as soon as possible, alert your client's users (if necessary), and have it removed from the list of featured clients if it won't be fixed in a reasonable amount of time.
+
+
+### Feature Parity
+To be featured, your client must have at least these features. 
+
+- Sending, editing, and deleting posts
+- Updating chats when `update_chat` is sent (or similar in the case of updates) 
+- User profiles
+	- Quotes
+- Profile picture support
```

Comment:
custom profile support? oh, the feature that svelte doesn't have for like 2 months already? (sorry for being passive-agressive, but this is just hypocrisy at this point)

custom profiles shouldn't be a requirement, like in meower95, they would break any sort of aesthetic/feel to it, especially with the 16x16 size. it also isn't easily supported everywhere, especially with most of them being webps.

y'all know not every client is web, right?

Reply:
> custom profile support? oh, the feature that svelte doesn't have for like 2 months already? (sorry for being passive-agressive, but this is just hypocrisy at this point)
> 
> custom profiles shouldn't be a requirement, like in meower95, they would break any sort of aesthetic/feel to it, especially with the 16x16 size. it also isn't easily supported everywhere, especially with most of them being webps.
> 
> y'all know not every client is web, right?

for 1, svelte is now just called client. 2 it is being worked on privately.

Reply:
why are they "absolutely required"? again, gatekeeping for no absolute reason.
alright, the 34 preset avatars are an alright and simple requirement, even with 2 text characters you can represent them. but enforcing custom avatars just make zero sense. what about text based clients? what about non-web clients? what about clients that don't use custom avatars for an aesthetic reason?